### PR TITLE
Timespinner: Update AP to have parity with standalone options

### PR DESCRIPTION
**Pull request by Ehseezed**
## What is this fixing or adding?
Adds in the `Prism Break` and `Aura Cap` options, as well as introducing the `Throw Stun Trap` item to the trap pool.
Also changed "mini jackpot" room to "jacksquat" room because of community feedback during an event.

## How was this tested?
Tested using debug `!getitem` as well as a complete test game with no issues found with the functionality.

### CODE REVIEW

File: worlds/timespinner/Options.py [hunk 1]
```diff
@@ -359,13 +366,18 @@ class TrapChance(Range):
 class Traps(OptionList):
     """List of traps that may be in the item pool to find"""
     display_name = "Traps Types"
-    valid_keys = { "Meteor Sparrow Trap", "Poison Trap", "Chaos Trap", "Neurotoxin Trap", "Bee Trap" }
-    default = [ "Meteor Sparrow Trap", "Poison Trap", "Chaos Trap", "Neurotoxin Trap", "Bee Trap" ]
+    valid_keys = { "Meteor Sparrow Trap", "Poison Trap", "Chaos Trap", "Neurotoxin Trap", "Bee Trap", "Throw Stun Trap" }
+    default = [ "Meteor Sparrow Trap", "Poison Trap", "Chaos Trap", "Neurotoxin Trap", "Bee Trap", 'Throw Stun Trap' ]
```
Comment (Exempt-Medic) [inline worlds/timespinner/Options.py:370]:
```suggestion
    default = [ "Meteor Sparrow Trap", "Poison Trap", "Chaos Trap", "Neurotoxin Trap", "Bee Trap", "Throw Stun Trap" ]
```

File: worlds/timespinner/__init__.py [hunk 2]
```diff
@@ -101,6 +101,7 @@ def fill_slot_data(self) -> Dict[str, object]:
             "DamageRando": self.options.damage_rando.value,
             "DamageRandoOverrides": self.options.damage_rando_overrides.value,
             "HpCap": self.options.hp_cap.value,
+            "MpCap": self.options.aura_cap.value,
```
Comment (Jarno458) [inline worlds/timespinner/__init__.py:104]: this string must be `"AuraCap"` to be picked up by the client

File: worlds/timespinner/Options.py [hunk 3]
```diff
@@ -180,12 +180,19 @@ class DamageRandoOverrides(OptionDict):
...
 class HpCap(Range):
-    "Sets the number that Lunais's HP maxes out at."
+    """Sets the number that Lunais's HP maxes out at."""
     display_name = "HP Cap"
     range_start = 1
     range_end = 999
     default = 999
 
+class AuraCap(Range):
+    """Sets the number that Lunais's MP maxes out at."""
```
Comment (Jarno458) [inline worlds/timespinner/Options.py:190]: I prefer these descriptions to match the non ap version, so this should be 
`"""Sets the maximum Aura Lunais is allowed to have. Level 1 is 80. Djinn Inferno costs 45"""`

**Review by Jarno458 — CHANGES_REQUESTED** [review]
Add on line 226 in _init_.py
```
        elif name in {"Laser Access A", "Laser Access I", "Laser Access M"} \
                and not self.options.prism_break:
            item.classification = ItemClassification.filler
```

**Review by Jarno458 — APPROVED** [review]
LGTM, ran it through some test generations and all seems fine

File: worlds/timespinner/LogicExtensions.py [hunk 4]
```diff
@@ -92,7 +93,10 @@ def can_break_walls(self, state: CollectionState) -> bool:
             return True
 
     def can_kill_all_3_bosses(self, state: CollectionState) -> bool:
-        return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
+        if self.flag_prism_break:
+            return state.has_all({'Laser Access M', 'Laser Access I', 'Laser Access A'}, self.player)
+        else:
+            return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
```
Comment (Exempt-Medic) [inline worlds/timespinner/LogicExtensions.py:99]: You can do this instead
```suggestion
        if self.flag_prism_break:
            return state.has_all({'Laser Access M', 'Laser Access I', 'Laser Access A'}, self.player)
        return state.has_all({'Killed Maw', 'Killed Twins', 'Killed Aelana'}, self.player)
```

File: worlds/timespinner/Options.py [hunk 5]
```diff
@@ -180,12 +180,19 @@ class DamageRandoOverrides(OptionDict):
...
 class HpCap(Range):
-    "Sets the number that Lunais's HP maxes out at."
+    """Sets the number that Lunais's HP maxes out at."""
     display_name = "HP Cap"
     range_start = 1
     range_end = 999
     default = 999
 
+class AuraCap(Range):
+    """Sets the maximum Aura Lunais is allowed to have. Level 1 is 80. Djinn Inferno costs 45"""
```
Comment (Exempt-Medic) [inline worlds/timespinner/Options.py:190]: Overall it's very inconsistent in this file, but this should probably have a period
```suggestion
    """Sets the maximum Aura Lunais is allowed to have. Level 1 is 80. Djinn Inferno costs 45."""
```

**Review by Exempt-Medic — APPROVED** [review]
The changes look fine, and are consistent with the rest of the formatting in the files. Generations still work and the WebHost displays the new options well. I don't know the game well, but Jarno approved it so like, yeah

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**